src/Features/CSharp/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -121,6 +121,13 @@ public override bool IsTriggerCharacter(SourceText text, int characterPosition,
 
         private INamedTypeSymbol GetCompletionListType(ITypeSymbol type, INamedTypeSymbol within, Compilation compilation)
         {
+            // PERF: None of the SpecialTypes include <completionlist> tags,
+            // so we don't even need to load the documentation.
```

Comment:
nice. 

for others, can this be cached somehow? are we currently load or parse those xml repeatedly on every compilation change?